xchange-okcoin/src/main/java/info/bitrich/xchangestream/okcoin/OkExStreamingExchange.java

```diff
@@ -9,4 +9,8 @@
     public OkExStreamingExchange() {
         super(new OkCoinStreamingService(API_URI));
     }
+
+    public OkExStreamingExchange(String api_url) {
```

Comment:
please, use camelCase as it's default for java as well as for this project

Reply:
Code has been reformatted according to the standart, thank you for your advice.